3.6.4 release notes and KC updates

victoriaatraft opened the pull request:
## Summary of Changes
_Provide a brief summary of changes_
Pull request closes #3181 

Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/
and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html

<img width="1725" alt="Screenshot 2024-10-07 at 12 51 25 PM" src="https://github.com/user-attachments/assets/9d4a3d04-4ca7-41a7-8890-cce54d588f72">
<img width="1701" alt="Screenshot 2024-10-07 at 12 50 58 PM" src="https://github.com/user-attachments/assets/594b0e9a-55ab-4fd2-a351-6a5328202375">

victoriaatraft committed:
Release notes 3.6.4 and Updates to KC

Added 3.6.4 updates and Interpreting Error Types under Understanding Error Reports and File Structure sub page

victoriaatraft committed:
list correction

reitermb reviewed (approved):
💯 🚀

klinkoberstar commented:
Thanks @victoriaatraft and @reitermb ! Here are my comments:

- The following line may be redundant given the addition of the table to the KC: _Inconsistency across one or more cases in a file will trigger this status, where at least some of the records in the file weren't added to the database._
- @lfrohlich  raised a good point that the excel interface will vary for different users which we should resolve before release. Perhaps we can use general language or direct users to the Support Center for filtering in Excel.

ADPennington committed:
Merge branch 'develop' into 3.6.4-release-notes

ADPennington commented:
@victoriaatraft is there a gitbook version of the KC changes in this PR? I saw the release notes version (and shared some suggestions), but I have some language suggestions for the other KC updates too.

victoriaatraft commented:
> @victoriaatraft is there a gitbook version of the KC changes in this PR? I saw the release notes version (and shared some suggestions), but I have some language suggestions for the other KC updates too.

@ADPennington hi here is the [Gitbook link for the Knowledge Center](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/knowledge-center-drafts/interpreting-error-types). The error type table and red text is what is new content on the Understanding Error Reports and File Structure page.

klinkoberstar commented:
> > @victoriaatraft is there a gitbook version of the KC changes in this PR? I saw the release notes version (and shared some suggestions), but I have some language suggestions for the other KC updates too.
> 
> @ADPennington hi here is the [Gitbook link for the Knowledge Center](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/knowledge-center-drafts/interpreting-error-types). The error type table and red text is what is new content on the Understanding Error Reports and File Structure page.

Thank you for linking this @victoriaatraft - so much easier to make comments here! I made some additional suggested edits and left comments to give some context about why I proposed changes. I've been in the weeds with this content for a bit though so open to suggestions about how best to share this info.

victoriaatraft committed:
Release Notes Updates and KC

3.6.4 release notes added fical/calendar table and tightened content for interpreting error types table

victoriaatraft committed:
Typo

victoriaatraft committed:
3.6.4 updates

KC and release notes

ADPennington commented on product-updates/knowledge-center/index.html:
```diff
@@ -337,6 +337,151 @@ <h2 class="usa-card__heading" id="existing">Use an existing Login.gov Account</h
+                Fiscal Quarter Table Explainer to the TDP Data Files Page
+              </h5>
+              <p>The records within your data files are organized by <b>calendar quarter period</b> which runs from January 1 to December 31. However, these files must be reported based on the fiscal quarter period, which runs from October 1 to September 30. For example, files that cover <b>Calendar Year 2024 Quarter 4</b> are to be reported for <b>Fiscal Year 2025 Quarter 1</b>, which is due by  ...
+              <p>We've <b>added</b> the table below to the <a href="https://tanfdata.acf.hhs.gov/data-files">data files submission page</a> to help users map the calendar period content within the file to the reporting fiscal period.</p>  
```
sorry the confusion but i thought this link would point to the KC page https://tdp-project-updates.app.cloud.gov/knowledge-center/uploading-data.html and that this page would be updated with how the page will look? 

And something i'm also just realizing -- was this change actually made to TDP yet? I dont recall approving a dev ticket for this change, but i did approve the design ticket related to this. 

If this change isnt actually included in a qasp-approved dev ticket, then we should table this note for a future release. @victoriaatraft @reitermb @klinkoberstar @lfrohlich

victoriaatraft replied:
resolved on our call - content with this is moved for [future release notes](https://app.gitbook.com/o/Yd4Wv0Fi89kSKpmakWrn/s/uAlDVLhlFIbJw6huUS2x/release-notes-workflow/future-release-notes) on gitbook.

victoriaatraft committed:
3.6.4 updates

victoriaatraft committed:
KC Impact column

victoriaatraft commented:
@ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.

Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/
and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html

ADPennington commented:
> @ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.
> 
> Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/ and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html

@victoriaatraft couple of followups:

- i think the impact column in the release notes is different from the impact column in the KC. I think we want the version in the KC? 
- do we want to move up the case consistency description in the KC? it appears last in the descriptions but 2nd in the table.

victoriaatraft committed:
3.6.4 table and KC updates

victoriaatraft commented:
> > @ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.
> > Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/ and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html
> 
> @victoriaatraft couple of followups:
> 
> * i think the impact column in the release notes is different from the impact column in the KC. I think we want the version in the KC?
> * do we want to move up the case consistency description in the KC? it appears last in the descriptions but 2nd in the table.

@ADPennington agree. done 😊

ADPennington commented:
> > > @ADPennington @lfrohlich @klinkoberstar edits have been made per our meeting this afternoon.
> > > Changes reviewable at: https://reitermb.github.io/Test/knowledge-center/ and: https://reitermb.github.io/Test/knowledge-center/viewing-error-reports.html
> > 
> > 
> > @victoriaatraft couple of followups:
> > 
> > * i think the impact column in the release notes is different from the impact column in the KC. I think we want the version in the KC?
> > * do we want to move up the case consistency description in the KC? it appears last in the descriptions but 2nd in the table.
> 
> @ADPennington agree. done 😊

@victoriaatraft thank you :) i did a quick comparison of the live KC (left) and the test KC (right), and some things moved around. Please see below:

![Screenshot 2024-10-09 114306](https://github.com/user-attachments/assets/2dba2841-22a3-4431-9b29-1703540601b6)
![Screenshot 2024-10-09 114033](https://github.com/user-attachments/assets/1feca645-fb95-439e-9e0f-40fa6f318f8b)
![Screenshot 2024-10-09 113923](https://github.com/user-attachments/assets/1f17f9ad-62e3-4fb9-b310-dfcae53c951c)

klinkoberstar commented:
Good catch by Alex on the example section - otherwise, the language looks good to me!

victoriaatraft committed:
3.6.4 table and Common Errors KC

fixed icon spacing for release notes and ordering of Examples of Common Errors in the KC

victoriaatraft committed:
a11y list issue in index

victoriaatraft committed:
KC updates viewing error reports

Commas added

ADPennington commented:
@victoriaatraft -- @klinkoberstar and I just took a final sweep of the release notes and KC updates and have a couple of final suggestions. Please ping us when these changes are pushed up to the test environment for our final review. cc: @lfrohlich @ttran-hub

- please change the release note date to Oct 10, 2024. I have completed testing of the release in staging and believe we will push the release to prod some time today. 
![Screenshot 2024-10-10 123622](https://github.com/user-attachments/assets/d2e1e3d4-484f-49e5-833d-905a56958b8e)


- Can we move the text referenced below above the table (in a new paragraph)?
![Screenshot 2024-10-10 123553](https://github.com/user-attachments/assets/65f7c3ca-d588-434b-bae5-cfcd54992273)

victoriaatraft committed:
3.6.4 date and kc change

victoriaatraft commented:
@klinkoberstar @ADPennington done
cc: @lfrohlich @ttran-hub

ADPennington reviewed (approved):
LGTM 🥇 cc: @lfrohlich @ttran-hub
